ModelBundle/Manager/EntityDbMapper.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Manager;
+
+use OpenOrchestra\ModelInterface\Manager\EntityDbMapperInterface;
+use Doctrine\ODM\MongoDB\DocumentManager;
+use Doctrine\ODM\MongoDB\Persisters\PersistenceBuilder;
+
+class EntityDbMapper implements EntityDbMapperInterface
```

Comment:
I'm not agree as EntityToDbMapper is in Manager directory and make functions as doctrine ObjectManager, @nicolasThal